store/cachekv/internal/mergeiterator.go

```diff
@@ -137,11 +135,12 @@ func (iter *cacheMergeIterator) Value() []byte {
 
 // Close implements Iterator
 func (iter *cacheMergeIterator) Close() error {
+	err1 := iter.cache.Close()
```

Comment:
should we early return if cache Close err?

Reply:
I think it's important to close both iter, because the close could release some system resource or do unlock.